python-avd/pyavd/_eos_designs/shared_utils/mlag.py

```diff
@@ -77,17 +87,23 @@ def mlag_peer_l3_ipv6_pool(self: SharedUtilsProtocol) -> str:
     @cached_property
     def mlag_role(self: SharedUtilsProtocol) -> Literal["primary", "secondary"] | None:
         # Note: self.node_group_is_primary_and_peer_hostname is always set when self.mlag is true, so this is just to make type-checker happy.
```

Comment:
I think the comment is not correct anymore right?

Reply:
right.
```suggestion
```